File: Minecraft-server/src/main/java/org/example/commands/Greet.java
package org.example.commands;

import net.minestom.server.command.builder.Command;
import net.minestom.server.entity.Player;

public class Greet extends Command {
    public Greet() {
        super("greet");
        addSyntax((sender, _) -> {
            if (sender instanceof Player) sender.sendMessage("Hello " + ((Player) sender).getUsername());
        });
    }
}

File: Minecraft-server/src/main/java/org/example/commands/Greet.kt
package org.example.commands

import net.minestom.server.entity.Player
import revxrsal.commands.annotation.Command
import revxrsal.commands.minestom.actor.MinestomCommandActor

class Greet {
    @Command("greet")
    fun greet(actor: MinestomCommandActor, target: Player){
        actor.reply("Hello, ${target.name}!")
    }
}